cdap-common/src/main/java/io/cdap/cdap/common/http/CDAPHttpHeaderNames.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright © 2022 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.cdap.cdap.common.http;
+
+/**
+ * Names for any Http headers introduced by CDAP should be placed here.
+ * This will help keeping the context about headers in one place.
+ */
+public class CDAPHttpHeaderNames {
```

Comment:
The code is inside CDAP, no need to have `CDAP` again as part of the class name. Also, make this a final class with a private constructor to prevent instantiation. Since we are introducing this class, we should move any header constants to here as well.

Reply:
Renamed the file. I am not familiar with all other headers being used. Created a JIRA on myself to do this in another PR https://cdap.atlassian.net/browse/CDAP-18799